.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: ci
```

Comment:
Change to `Continuous Integration`

Reply:
Just FYI, we have limited number of free `GitHub Workflow` runs per month.

We've other repositories, which also are using GitHub Actions. Not sure how intensive you are planning to use them, but keep in mind the above.

Reply:
Not much, just for simple linting and security checks.